scripts/functions.sh

```diff
@@ -127,3 +127,65 @@ function askSelection {
         dialogREPLY="$REPLY"
     fi
 }
+
+function detectBoard {
+
+    BOARD_MODEL="Unknown"
+    BOARD_NAME="Unknown"
+    BOARD_OS="Unknown"
```

Comment:
Are these variables exported?

Reply:
No.

Instead of being exported  somewhere in the script then used by the modules, the function is part of a shared file and the function  can be called by any module when needed to set it in their environment. 

This allows each module to be run without having to run the whole script to set them and be independent.

Reply:
Ah I see, so the variables are set every time.